staging/src/k8s.io/kubectl/pkg/cmd/delete/delete.go

```diff
@@ -257,6 +259,9 @@ func (o *DeleteOptions) Validate() error {
 	}
 
 	if len(o.Raw) > 0 {
+		if o.Interactive {
+			return fmt.Errorf("--interactive can not be used with --raw")
+		}
```

Comment:
Can we add a test for this validation? I don't see anything similar for delete, but Apply has flag validation tests here: https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/kubectl/pkg/cmd/apply/apply_test.go#L100